odoo/addons/base/models/ir_actions_report.py

```diff
@@ -554,7 +555,7 @@ def barcode(self, barcode_type, value, **kwargs):
             barcode_type = 'Code128'
 
         try:
-            barcode = createBarcodeDrawing(barcode_type, value=value, format='png', **kwargs)
+            barcode = createBarcodeDrawing(barcode_type, value=value, **kwargs)
```

Comment:
To reviewers: the `format='png'` kwarg isn't used by `createBarcodeDrawing`. It's useless to have it here.
It's here since many years ago, when `createBarcodeImageInMemory` was used instead.